int32 num_detections
Detection3D[] detections
